src/overlays/actors/ovl_En_Ishi/z_en_ishi.h

```diff
@@ -6,26 +6,40 @@
 struct EnIshi;
 
 typedef void (*EnIshiActionFunc)(struct EnIshi*, PlayState*);
-typedef void (*EnIshiUnkFunc)(struct EnIshi*, PlayState*);
-typedef void (*EnIshiUnkFunc2)(Actor*, PlayState*);
-
-#define ENISHI_GET_1(thisx) ((thisx)->params & 1)
-#define ENISHI_GET_2(thisx) (((thisx)->params >> 1) & 1)
-#define ENISHI_GET_4(thisx) (((thisx)->params >> 2) & 1)
-#define ENISHI_GET_8(thisx) (((thisx)->params >> 3) & 1)
-#define ENISHI_GET_70(thisx) (((thisx)->params >> 4) & 7)
-#define ENISHI_GET_F0(thisx) (((thisx)->params >> 4) & 0xF)
-#define ENISHI_GET_100(thisx) (((thisx)->params >> 8) & 1)
-#define ENISHI_GET_FLAG(thisx) (((thisx)->params >> 9) & 0x7F)
+// used for dust spawn and draw funcs
+typedef void (*EnIshiMultiFunc)(struct EnIshi*, PlayState*);
+typedef void (*EnIshiSpawnDebrisFunc)(Actor*, PlayState*);
+
+// silver boulder instead of small rock
+#define ENISHI_GET_BIG_FLAG(thisx) ((thisx)->params & 1)
+#define ENISHI_GET_SPAWN_BUGS_FLAG(thisx) (((thisx)->params >> 1) & 1)
+// for sitting still on walls in TF/Ikana Graveyard
+#define ENISHI_GET_IGNORE_SNAP_TO_FLOOR(thisx) (((thisx)->params >> 2) & 1)
+// dual objects: this actor can use field_keep or object_ishi
+// the only place its used in u0 is the swamp spiderhouse, which uses dangeon_keep
```

Comment:
`u0`? Us version 0?

Reply:
Yeah that's what I meant. I haven't seen any work on the Japanese version yet, but I thought I heard it was quite different, and it may have been used there for something else. 

At some point, documentation from the perspective of U0 only might be too limited.

Reply:
Theres only one US version, so it seems better to me to just "the US version"